services/deployment-agent/Makefile

```diff
@@ -64,6 +64,9 @@ build build-kit build-x build-devel build-devel-kit build-devel-x: ## Builds $(A
 up: .init ${DEPLOYMENT_AGENT_CONFIG} ${TEMP_COMPOSE} ## Deploys or updates current stack "$(STACK_NAME)" using replicas=X (defaults to 1)
 	@docker stack deploy -c ${TEMP_COMPOSE} $(STACK_NAME)
 
+up-aws: .init ${DEPLOYMENT_AGENT_CONFIG} ${TEMP_COMPOSE}-aws ## Deploys or updates current stack "$(STACK_NAME)" using replicas=X (defaults to 1)
```

Comment:
Comparing ``up-aws`` with ``up-devel``, I wonder what is the different (or the correct one) between
```
${TEMP_COMPOSE}-aws
${TEMP_COMPOSE-aws}
```

Reply:
To follow the convention, the last one is the good one. I corrected